### DISCUSSION TITLE
adding a new features

### DISCUSSION THREAD

**Discussion by parsman** [pterodactyl/panel#3241] [Feature Requests]
Hello
I had a suggestion or request, please add a new feature to monitoring the game servers Like [LGSL](https://github.com/tltneon/lgsl)
I think this feature is needed more than anything...

**Comment by FoksVHox** [comment]
Why is this feature needed? There are more important stuff that should be implemented first, like #279, #1219 & #2234

**Reply by parsman** [reply]
Of course, these things that you said have more priority... Especially audit logs!

**Reply by FoksVHox** [reply]
You just said that "I think this feature is more needed than anything" - which indicates that in your opinion that should have a critical priority?
